examples/favorites_advanced/getx_favorites_advanced/pubspec.yaml

```diff
@@ -18,18 +18,20 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.9.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
 
-  appbar_textfield: ^1.0.1+1
-  badges: ^1.2.0
-  curved_navigation_bar: ^0.3.7
-  get: ^3.26.0
-  get_instance: ^3.13.0
-
+  appbar_textfield: ^2.0.0
+  badges: ^2.0.0-nullsafety.1
+  curved_navigation_bar: ^0.4.0-nullsafety.0
+  get: ^4.1.1
+  rxdart: ^0.26.0
+  favorites_advanced_base:
+  mockito: ^5.0.3
```

Comment:
@RomanovaPrime, are you sure you need mockito as a dependency? To mee looks like we need it as `dev_dependencies` .

Reply:
I found it here https://pub.dev/packages/mockito/install